v3/src/components/graph/components/graph-axis.tsx

```diff
@@ -35,7 +39,7 @@ export const GraphAxis = observer((
           getAxisModel={() => graphModel.getAxis(place)}
           label={getLabel()}
           enableAnimation={enableAnimation}
-          showScatterPlotGridLines={graphModel.plotType === 'scatterPlot'}
+          showScatterPlotGridLines={graphModel.plotType === 'scatterPlot' && ['left', 'bottom'].includes(place)}
```

Comment:
Based on our conversation earlier today, a property like `canShowScatterPlotGridLines` could be a configuration property.

Reply:
Done